monkey/monkey_island/cc/resources/auth/register.py

```diff
@@ -27,10 +28,17 @@ def post(self):
         Registers a new user using flask security register
 
         """
-        response: ResponseValue = register()
+        try:
+            username, password = get_username_password_from_request(request)
+            response: ResponseValue = register()
+        except Exception:
+            return bad_request_response()
+
+        if isinstance(response, str):
```

Comment:
Also, maybe test if it's NOT what we expect rather than if it's a string. There might be other formats that flask-security-too returns?